ScoutSuite/providers/base/configs/resources.py

```diff
@@ -8,3 +8,27 @@ class Resources(dict, metaclass=abc.ABCMeta):
     @abc.abstractmethod
     async def fetch_all(self, **kwargs):
         raise NotImplementedError()
+
+
+class SimpleResources(Resources, metaclass=abc.ABCMeta):
+
```

Comment:
Would you mind adding docstrings to SimpleResource and CompositeResource?

Reply:
Done! Feel free to modify them, I did my best to be concise and clear but they may be still confused!